$NetBSD$

Don't unconditionally reset LDFLAGS and CPPFLAGS because pkgsrc passes
important settings in those.

Change test ... == to test ... = for portability.

--- configure.orig	2009-03-02 17:59:25.000000000 +0000
+++ configure
@@ -4190,10 +4190,10 @@ else
 fi
 
 
-LDFLAGS="-L/usr/X11R6/lib -L/usr/X11R6"
-CPPFLAGS="-I/usr/X11R6/include -I/usr/X11R6"
+#LDFLAGS="-L/usr/X11R6/lib -L/usr/X11R6"
+#CPPFLAGS="-I/usr/X11R6/include -I/usr/X11R6"
 
-if test `uname` == Darwin; then
+if test `uname` = Darwin; then
 LDFLAGS="$LDFLAGS -lmx"
 CPPFLAGS="$CPPFLAGS -DFFTW_USE_DOUBLE -D__DARWIN__ -D__MACOSX__"
 else
@@ -4353,10 +4353,10 @@ echo "$as_me: error: *** Can't find the 
    { (exit 1); exit 1; }; }
 fi
 
-if test `uname -s` == FreeBSD; then
+if test `uname -s` = FreeBSD; then
     GL_LIBS="-lGL -lGLU"
 else
-    if test `uname -s` == Darwin; then
+    if test `uname -s` = Darwin; then
         GL_LIBS="-framework OpenGL"
     else
         GL_LIBS="-lGL -lGLU -lpthread"
@@ -4943,7 +4943,7 @@ fi
     OGG_LIBS="-L$prefix/lib"
   fi
 
-if test `uname` == Darwin; then
+if test `uname` = Darwin; then
   OGG_LIBS="$OGG_LIBS -framework Ogg"
 else
   OGG_LIBS="$OGG_LIBS -logg"
@@ -5073,7 +5073,7 @@ fi
     VORBIS_LIBS="-L$prefix/lib"
   fi
 
-  if test `uname` == Darwin; then
+  if test `uname` = Darwin; then
     VORBIS_LIBS="$VORBIS_LIBS -framework Vorbis -lm"
     VORBISFILE_LIBS=""
     VORBISENC_LIBS=""
@@ -5702,7 +5702,7 @@ fi
     FFTW_LIBS="-L$prefix/lib"
   fi
 
-if test `uname` == Darwin; then
+if test `uname` = Darwin; then
   FFTW_LIBS="$FFTW_LIBS -framework FFTW3"
 else
   FFTW_LIBS="$FFTW_LIBS -lfftw3f"
@@ -6323,7 +6323,7 @@ echo "$as_me: error: *** Can't find the 
    { (exit 1); exit 1; }; }
 fi
 
-if test `uname` == Darwin; then
+if test `uname` = Darwin; then
 SDL_LIBS="$SDL_LIBS -framework SDL_net"
 else
 SDL_LIBS="$SDL_LIBS -lSDL_net"
@@ -7445,7 +7445,7 @@ done
 
 
 # FIXEME
-if test `uname` == Darwin; then
+if test `uname` = Darwin; then
 LIBS="$LIBS /usr/X11R6/lib/libexpat.a -lz -framework UnixImageIO"
 else
 LIBS="$LIBS -lexpat -lz -lpng -ljpeg"
@@ -7466,7 +7466,7 @@ fi
 
 
 
-if test `uname` == Darwin; then
+if test `uname` = Darwin; then
 prefix=./
 fi
 
